aviatrix/resource_aviatrix_dcf_ips_profile_vpc.go

```diff
@@ -100,3 +106,11 @@ func resourceAviatrixDCFIpsProfileVpcDelete(ctx context.Context, d *schema.Resou
 	d.SetId("")
 	return nil
 }
+
+func isVpcNotFoundError(err error) bool {
```

Comment:
We should use a designated AVXERR here if we need to know that something specific happened. Make a follow-up jira to fix and add a note here with the ID.